packages/salesforcedx-vscode-core/src/commands/forceApexLogGet.ts

```diff
@@ -90,9 +93,8 @@ export class ForceApexLogGetExecutor extends SfdxCommandletExecutor<
         mkdir('-p', logDir);
       }
 
-      const date = moment(new Date(response.data.startTime)).format(
-        'YYYYMMDDhhmmss'
-      );
+      const localUTCDate = new Date(response.data.startTime);
+      const date = getYYYYMMddHHmmssDateFormat(localUTCDate);
```

Comment:
This is used in naming the debug log that gets downloaded when running `SFDX: Get Debug Logs`